README.md

```diff
@@ -102,45 +102,54 @@ Make a copy of your access key ID and secret access key. You'll need to include
     npm run db:migrate -- up
     ```
 
-13. Run the server:
+13. Seed your development database with test member and project data:
+    ```bash
+    npm run db:copy -- <STATE>
+    ```
+
+    Available `STATE` options:
+    - `GOAL_SELECTION` (default)
+    - `GOAL_SELECTION_VOTES`
+    - `PRACTICE`
+    - `REFLECTION`
+
+14. Run the server:
 
     ```bash
     npm start
     ```
 
-14. Visit the server in your browser:
+15. Visit the server in your browser:
 
     ```bash
     open http://idm.learnersguild.dev
     ```
 
-15. Create an IDM Account
+16. (OPTIONAL) Add some test users:
 
-    Go to `localhost:8080` and use the `Data Explorer` to run the following command to issue yourself an invitation code.
-
-    ```ReQl
-    r.db('idm_development').table('inviteCodes').insert({
-      id: '58abd2aa-3826-4604-bf7c-f8f2cf7eaad9',
-      code: 'hand_crafted_artisanal_invite_code',
-      description: 'hand crafted artisanal invite code',
-      roles: ['admin', 'member'],
-      active: true,
-      createdAt: r.now(),
-      updatedAt: r.now(),
-    })
+    ```bash
+    npm run data:users -- --verbose --role=member some-dummy-invite-code
     ```
 
-16. Sign In
+## Gotchas
 
-    Use the invitation code `hand_crafted_artisanal_invite_code` to create an account.
+### Create an IDM Account
 
-17. (OPTIONAL) Add some test users:
+Go to `localhost:8080` and use the `Data Explorer` to run the following command to issue yourself an invitation code.
```

Comment:
If framed as a "gotcha", let's add this sentence before this:

If you've used the `db:copy` script to seed the database and still aren't logged in, there isn't an idm user in the test data linked to your github account. You'll need to manually insert an invite code to the database and use it to sign up in your local `idm` service.